scripts/release.py

```diff
@@ -0,0 +1,21 @@
+#!/bin/env python3
+import os
+import sys
+import shutil
+
+if len(sys.argv) != 2:
+    print("Usage: {} VERSION".format(sys.argv[0]))
+    sys.exit(1)
+
+version = str(sys.argv[1])
+
+releasePath = os.path.join(os.path.dirname(__file__), "..", "build_release")
+
+if os.path.exists(releasePath):
+    shutil.rmtree(releasePath)
+
+os.mkdir(releasePath)
+os.chdir(releasePath)
+os.system("cmake -DCPACK_PACKAGE_FILE_NAME=\"crow-{}\" .. && make -j5".format(version))
```

Comment:
Might be a good idea to disable examples and tests building since all we really need is crow_all.h
```suggestion
os.system("cmake -DBUILD_EXAMPLES=OFF -DBUILD_TESTING=OFF -DCPACK_PACKAGE_FILE_NAME=\"crow-{}\" .. && make -j5".format(version))
```

Reply:
Good catch, fixed.